File: Python/scraping/extract_images.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto("https://example.com")
    images = page.eval_on_selector_all("img", "els => els.map(el => el.src)")
    print(images)
    browser.close()

File: NodeJS/scraping/extract_images.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('https://example.com');
  const images = await page.$$eval('img', els => els.map(el => el.src));
  console.log(images);
  await browser.close();
})();
